xcube/core/resampling/rectify.py

```diff
@@ -32,12 +33,14 @@
 from .cf import maybe_encode_grid_mapping
 
 
+_INTERPOLATIONS = {"nearest": 0, "linear": 1, "bilinear": 2}
```

Comment:
Why not an enum? I don't insist on it, but it seems like an ideal use case for one.

Reply:
Will do.